blockly-core/README.md

```diff
@@ -1,7 +1,29 @@
-# Blockly Core
+# Blockly Core (Code Studio fork)
 
 This is a fork of [Blockly](https://code.google.com/p/blockly/), an open source visual programming environment.
 
+Major additions and changes in this fork:
```

Comment:
- support for a special UI for unattached blocks
- improved support for multiple blockspaces on a single page
- improvements to block arrangement on initialization